src/View/BottomPanel.java

```diff
@@ -0,0 +1,66 @@
+package View;
+
+import java.awt.FlowLayout;
+import java.awt.event.ActionEvent;
+import java.awt.event.ActionListener;
+
+import javax.swing.JButton;
+import javax.swing.JPanel;
+
+public class BottomPanel extends JPanel {
+	
+	private JButton menu;
+	private JButton build;
+	private JButton sell;
+	private JButton mortgage;
+	private JButton trade;
+	
+	private ButtonListener buttonListener;
```

Comment:
Again my first comment about Action Listeners applies Get rid of the Listeners functionality and we will just put the Actions in using lambdas.